prov/verbs/src/fi_verbs.c

```diff
@@ -2707,6 +2740,12 @@ fi_ibv_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 		goto err2;
 	}
 
+	ret = ibv_req_notify_cq(_cq->cq, 0);
+	if (ret) {
+		FI_WARN(&fi_ibv_prov, FI_LOG_CQ, "ibv_req_notify_cq failed\n");
+		goto err3;
+	}
```

Comment:
Move this into the switch statement where the completion channel has been created and configured.  Otherwise, this call will fail.